enet.pyx

```diff
@@ -819,6 +852,8 @@ cdef class Host:
 
     cdef ENetHost *_enet_host
     cdef bool dealloc
+    cdef object _interceptCallback
```

Comment:
ENetHost.intercept is a C function, and in pyenet we want to trigger a python function. We can't simply assign a python function to a variable that expects C function. That's why we store a reference to python function in ENetHost.